cmake/Modules/LAMMPSInterfacePlugin.cmake

```diff
@@ -163,12 +164,12 @@ if(BUILD_MPI)
 
       ExternalProject_get_property(mpi4win_build SOURCE_DIR)
       file(MAKE_DIRECTORY "${SOURCE_DIR}/include")
-      add_library(MPI::MPI_CXX UNKNOWN IMPORTED)
```

Comment:
We're a C++ code. The correct way to use the **C bindings** of MPI with C++ is to use the MPI_CXX target. I've spent a significant amount of time correcting this in other places. Don't go backwards.

Reply:
I understand what you think you want. But this is not it. You're fixing the wrong thing.